src/config/app.config.ts

```diff
@@ -0,0 +1,7 @@
+import { env } from 'process';
```

Comment:
fell free to delete empty file src/common/.gitkeep

unfortunately can't comment empty file. it's because this comment here